scripts/retrieve_prs.py

```diff
@@ -0,0 +1,145 @@
+import json
+import locale
+import os
+import re
+import sys
+import argparse
+import subprocess
+from collections import namedtuple
+from os.path import expanduser
+
+import requests
+
+Features = namedtuple(
+    "Features",
+    [
+        "title",
+        "pr_number",
+        "labels",
+    ],
+)
+
+
+def run(command):
+    """Returns (return-code, stdout, stderr)"""
+    p = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True)
+    output, err = p.communicate()
+    rc = p.returncode
+    enc = locale.getpreferredencoding()
+    output = output.decode(enc)
+    err = err.decode(enc)
+    return rc, output.strip(), err.strip()
```

Comment:
@carolineechen Looking at how the returned items are used, I think it is redundant to return `rc` and `err.strip()`. The error check can be directly performed in this function, then there is no need to return return code and stderr.

Also this can be simplified with the use of `check_output`. like the following (except ascii decoding part)

https://github.com/pytorch/audio/blob/184466a904c45d3a7e84629bff647b874f5448d5/setup.py#L15-L19